docs/swagger/api/discovery.swagger.json

```diff
@@ -0,0 +1,447 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "api/discovery.proto",
+    "version": "version not set"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {},
+  "definitions": {
+    "ATCPCheckerAction": {
+      "type": "object",
+      "properties": {
+        "send": {
+          "type": "string",
+          "format": "byte",
+          "description": "This is used to send a data along with a connection opening."
+        },
+        "expect": {
+          "type": "string",
+          "format": "byte",
+          "description": "Expecting content returned from the server."
+        }
+      },
+      "description": "Action represents a set of requests from Samaritan to the server\nand what the expected server returns."
+    },
+    "CompressionMethod": {
+      "type": "string",
+      "enum": [
+        "SNAPPY",
+        "MOCK"
+      ],
+      "default": "SNAPPY"
+    },
+    "EndpointState": {
+      "type": "string",
+      "enum": [
+        "UP",
+        "DOWN",
+        "UNKNOWN"
+      ],
+      "default": "UP",
+      "title": "- UP: healthy\n - DOWN: unhealthy\n - UNKNOWN: unknown"
+    },
+    "apiDependencyDiscoveryResponse": {
+      "type": "object",
+      "properties": {
+        "added": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/serviceService"
+          },
+          "description": "Added service."
+        },
+        "removed": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/serviceService"
+          },
+          "description": "Removed service."
+        }
+      }
+    },
+    "apiSvcConfigDiscoveryResponse": {
```

Comment:
1. How to use the generated file?
2. Most messages in `discovery.proto` won't present in the web api.

Reply:
like this
![image](https://user-images.githubusercontent.com/12208550/70426057-566efc80-1aad-11ea-89d3-fb77ebaa95ee.png)

Reply:
How about https://samaritan-proxy.github.io/docs/proto-ref/#config